app/grandchallenge/components/models.py

```diff
@@ -1143,6 +1151,15 @@ def _clean_store_in_database(self):
                 f"Interface {self.kind} objects cannot be stored in the database"
             )
 
+    def _validate_example_value(self):
```

Comment:
Please be consistent with the rest of the codebase. Method name should be `_clean_example_value`.

Reply:
Done